samples/bluetooth/direction_finding_connectionless_tx/sample.yaml

```
@@ -0,0 +1,10 @@
+sample:
+  name: Direction Finding Connectionless Beacon
+tests:
+  sample.bluetooth.df_beacon:
```

maybe update this to reflect new name:
```suggestion
  sample.bluetooth.direction_finding_connectionless:
```

---

corrected